NEWS.md

```diff
@@ -15,6 +15,7 @@ are not recorded in the data. (#589)
 
 - New vignette [Development Process](../articles/development_process.html) and improvements made to contribution vignettes (#765 & #758)
 
+- `derive_var_ontrtfl()` now has `new_var` parameter to support the derivation of ONTRxxFL and ONTRTwFL variables. (#721)
```

Comment:
...now has **_a_** `new_var`...

Reply:
Also please remove the period at the end of the sentence and put variables names such as `ONTRxxFL` in backticks to make them appear in code font.